ql/src/java/org/apache/hadoop/hive/ql/ddl/table/creation/ShowCreateTableOperation.java

```diff
@@ -50,20 +53,26 @@
 import org.apache.hive.common.util.HiveStringUtils;
 import org.stringtemplate.v4.ST;
 
+import com.google.common.collect.ImmutableSet;
+
+import avro.shaded.com.google.common.collect.Sets;
```

Comment:
Probably you did not want to use the shaded guava here.

Reply:
Thanks fixed.